developerportal/apps/events/templates/events.html

```diff
@@ -22,7 +22,7 @@
   {% endif %}
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.

Ignore for review